Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.yml

```diff
@@ -72,38 +72,44 @@ configuration:
   name: credentials
   type: 9
   required: false
+- additionalinfo: The API Root to use (e.g. default, public). If left empty, the server default API root will be used. When the server has no default root, the first available API root will be used instead.
+  display: API Root to use
```

Comment:
Should be consistent with capitalization in the display. I see that the display is usually all capitalized, so it should be API Root to Use